### PR TITLE
Remove unnecessary print statements

### LINKED ISSUE #612: Remove unnecessary/debugging uses of print from the codebase and examples
There are a number of places in the codebase and in the examples where the `print` function is used, and in some cases a commented out `print` statement. These need to be evaluated and removed.

### PR DESCRIPTION
**Pull request by rahulporuri**
fixes #612 

This PR removes unnecessary print statements and dead code/tests

### CODE REVIEW

File: chaco/tests/test_colormapper.py [hunk 1]
```diff
@@ -130,36 +130,3 @@ def test_no_alpha(self):
...
-##         grayscale_bins = array([0.0, 1.0])
-##         grayscale_steps = array([1])
-
-##         colormap = LinearSegmentedColormap(
-##             grayscale_colors, grayscale_bins, grayscale_steps
-##         )
-
-##         colormap._recalculate()
-
-##         print '**************', colormap._color_bands, colormap._value_bands
```
Comment (aaronayres35) [inline chaco/tests/test_colormapper.py:165]: I haven't looked at this test/relevant code closely yet, but just the word interpolation reminded me of https://github.com/enthought/enable/issues/20
I am going to see if this test passes with the latest enable

Comment (aaronayres35) [reply chaco/tests/test_colormapper.py:165]: ah never mind, a search of `LinearSegmentedColormap` in the codebase yields no results outside of this file... So these tests are definitely good to go

The test case should probably be renamed to either `ColormapTestCase` or `TestColormap`.

**Review by aaronayres35 — APPROVED** [review]
LGTM

Just one comment about renaming the test case in `test_colormapper.py`

File: chaco/tests/test_colormapper.py (hunk 1 above)
Comment (rahulporuri) [reply chaco/tests/test_colormapper.py:165]: renamed

**Pull request merged**